### ISSUE TITLE
ItemBlockGlowPanel doesn't check for a valid metadata before looking up the name

### ISSUE THREAD

**Issue by justinrusso** [aidancbrady/Mekanism#5012]
#### Issue description:
I am using Item Stages to stage the items, and attempting to do it via Wildcard. When doing this, the script errors. There may as well be more that do this, but this is the first item we were doing via wildcard that caused the script to error.

#### Steps to reproduce:
```
mods.ItemStages.addItemStage("test", <mekanism:glowpanel:*>);
```

#### Version:
**Minecraft:** 1.12.2
**Forge:** 14.23.3.2669
**Mekanism:** 1.12.2-9.4.3.330
**CraftTweaker:** 1.12-4.1.6
**MTLib:** 3.0.4
**Item Stages**: 1.0.34

#### Logs:

https://pastebin.com/xF9nWQVD

**Comment by thiakil** [comment]
Itemstages shouldnt be calling that function with the damage wildcard value

**Comment by thiakil** [comment]
Fixed in 108f4ddbe8d960c9f566af3330fee388c74dd400
